Homarus/static/identify.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
```

Comment:
You can delete this file.  Convert is the only endpoint/function you're exposing.  If we ever ended up exposing `ffprobe` in the future, we'd want to do something like this, though.